lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py

```diff
@@ -175,17 +181,29 @@
           volumeName: registryvolume
           disk:
             bus: virtio
-        - name: cloudinitdisk
-          volumeName: cloudinitvolume
+      volumes:
+        - name: registryvolume
+          registryDisk:
+            image: kubevirt/cirros-registry-disk-demo:latest
+
+- name: Start fedora vm with cloud init
+  kubevirt_vm:
+      state: running
+      wait: true
+      name: myvm
+      namespace: vms
+      memory: 1024M
+      cloud_init_nocloud:
```

Comment:
Looks ok, I wonder whether we could allow user only provide things like `username`, `password` and other configuration instead if providing base64 content?

Reply:
Of course, that's what userData is for :)

```yaml
...
cloud_init_nocloud:
        userData: |
          ssh-authorized-keys:
            - ssh-rsa AAAAB3NzaK8L93bWxnyp test@test.com
          password: fedora
          ... etc
```

Reply:
I can maybe add example of that rather then base64 example

Reply:
done